src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -8,21 +8,20 @@
 
 namespace Sentry.Unity
 {
+    internal static class UnitySdkInfo
+    {
+        public const string Version = "0.0.1-alpha";

```

Comment:
This one you can take from the version of the assembly. Like it's done in .NET

We version the assembly on Directory.Build.props so it'll automatically bump here